### PR TITLE
Upgrading version for august release.

### PR DESCRIPTION
**Pull request by hemanttanwar**
I missed the bot created version bump PR in July release. 
So taking of it now.

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -114,7 +114,7 @@ com.microsoft.azure:spring-data-cosmosdb;3.0.0-beta.1;3.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-identity;1.1.0
-unreleased_com.azure:azure-messaging-servicebus;7.0.0-beta.4
+unreleased_com.azure:azure-messaging-servicebus;7.0.0-beta.5
```
Comment (conniey) [inline eng/versioning/version_client.txt:117]: Who uses unreleased_servicebus?

File: sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Release History
+## 7.0.0-beta.5 (2020-08-11)
+- Added a short timeout of 1 second in between messages for sync receive only.
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md:3]:
```
- Remove public constructor for QueueDescription, TopicDescription, SubscriptionDescription.
- Expose CreateQueueOptions, CreateTopicOptions, CreateSubscriptionOptions to create entities.
- Flatten and remove MessageCountDetails in QueueRuntimeInfo, TopicRuntimeInfo, and SubscriptionRuntimeInfo.
- Limiting visibility of properties on QueueDescription, TopicDescription, SubscriptionDescription to only those that can be updatedd.
```

File: eng/versioning/version_client.txt (hunk 1 above)
Comment (JimSuplizio) [reply eng/versioning/version_client.txt:117]: It's used down below in sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-servicebus/pom.xml

Comment (hemanttanwar) [reply eng/versioning/version_client.txt:117]: used by sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-servicebus/pom.xml

**Pull request merged**